pharmacy-ui/src/views/Drug/Drug.scss

```diff
@@ -0,0 +1,22 @@
+.wrapper {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    grid-template-areas: 'sales drug'; 
+  }
+  
+  .box1 {
+    grid-column-start: 1;
```

Comment:
you can set `grid-area: sales` or `grid-area: drug` instead of having to define explicit row/column start/end

Reply:
Done.